runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapPlugin.java

```diff
@@ -114,6 +128,15 @@ protected void startUp() throws Exception
 		updateShownIcons();
 	}
 
+	@Subscribe
+	private void onWidgetLoaded(WidgetLoaded l)
+	{
+		if (l.getGroupId() == WidgetID.WORLD_MAP_GROUP_ID)
+		{
+			updateQuestStartPointIcons();
```

Comment:
Why is this required?

Reply:
I got this from the last PR, no idea. Maybe @Rens-br can explain?

Reply:
This won't work probably generally since startUp tends to fire before anyone is ever logged in. Hooking to the map widget load like before is probably better since it also handles whenever you change accounts

Reply:
I can't remember fully but I think it was so it only updates when the worldmap is loaded instead of any widget